tools/format_python_tools.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+VENV_DIR="pyformat"
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+cd "$SCRIPTPATH"
+
+if [ "${CIRCLECI}" != "true" ]; then
```

Comment:
Is this because we want to avoid network traffic on CircleCI? Ideally we have it as consistent with local as possible.

Reply:
This is because in CircleCI installation of the dependencies happens as a [run stage](https://github.com/envoyproxy/envoy/pull/4882/files#diff-1d37e48f9ceff6d8030570cd36286a61R99)